### ISSUE TITLE
USER ISSUE: bug for claim stake

### ISSUE THREAD

**Issue by Gefferzzz** [StrangeLoopGames/EcoIssues#24863]
\
**User Id: (Not logged user)**
**Version:** 0.10.2.0 beta release-640

**Steps to Reproduce:**
add 100 claim papers to claim stake with 155 already in

**Expected behavior:**
just added plots to add

**Actual behavior:**
The following error message popped up:
```
Server encountered an exception:

<size=60.00%>Exception: NullReferenceException
Message:Object reference not set to an instance of an object.
Source:Eco.Gameplay

System.NullReferenceException: Object reference not set to an instance of an object.
   at Eco.Gameplay.Components.PlotsComponent.DescribePlotsClaimed()
   at InvokeStub_PlotsComponent.DescribePlotsClaimed(Object, Object, IntPtr*)
   at System.Reflection.MethodInvoker.Invoke(Object obj, IntPtr* args, BindingFlags invokeAttr)
Outer Exceptions:
Exception has been thrown by the target of an invocation.
Exception has been thrown by the target of an invocation.</size>

```

**Do you have mods installed? Does the issue happen when no mods are installed?:**
nope

**Please provide log files. Instruction is here https://github.com/StrangeLoopGames/EcoIssues/wiki/Getting-the-Eco-game-log-file

**Comment by padmanek** [comment]
Did you spawn the papers? I managed to get the same exact error with spawned claim papers.

Steps to reproduce:
1. Spawn some homestead support papers
2. Put them inside homestead stake (works even with global federation pre-setup)
3. Mouse over the 0/16 plots counter inside the stake.
4. client disconnected with above error

**Comment by D3nnis3n** [comment]
Please add your savegame to the report, you can find .db and .eco file in /Steam/steamapps/common/Eco/Eco_Data/Server/Storage.

**Comment by D3nnis3n** [comment]
Fixed in 10.2.1.
